### DISCUSSION TITLE
Has hipdnn been abandoned?

### DISCUSSION THREAD

**Discussion by Fourish** [ROCm/ROCm#3559] [Q&A] [answered]
Has hipdnn been abandoned? But why does hipify still use hipdnn in its documentation?

**Comment by saadrahim** [comment, marked as answer]
Thanks for the feedback. We should deprecate the hipdnn conversion capabilities. @Danny213123 please follow up on this.

Please look at [MIOpen](https://rocmdocs.amd.com/projects/MIOpen/en/latest/)  for equivalent functionality. hipDNN is no longer maintained.
